gdal-src/Cargo.toml

```diff
@@ -9,7 +9,7 @@ links = "gdal_src"
 [dependencies]
 link-cplusplus = "1.0"
 proj-sys = { version = "0.24.0", features = ["bundled_proj"] }
-libsqlite3-sys = { version = "0.28.0", features = ["bundled"], optional = true }
+libsqlite3-sys = { version = "0.30.1", features = ["bundled"], optional = true }
```

Comment:
That requires updating libsqlite3-sys in proj-sys as well

Reply:
I filled https://github.com/georust/proj/pull/209 for that. I also would suggest to use the same version range approach here, as libsqlite3-sys is a rather popular crate that is used by different crates with different update speeds.